### PR TITLE
Add Event Tracking to Academy Callouts

### PR DESCRIPTION
**Pull request by richbachman**
**Description of the change**:
I've set up a dataLayer function that fires when the Go to Course link is clicked. This sends Google Analytics event tracking information to the datalayer which gets sent to Google Tag Manager. So we should start seeing events in GA.

The event is structured in the following format:

event: 'customEvent',
eventCategory: 'Academy Callout',
eventAction: 'Click',
eventLabel: 'Go to Course',
eventValue: RISE URL 

We were also double counting Google Analytics traffic since that is being loaded on the docs site twice. I've removed the hard-coded GA number so it is only being loaded via Google Tag Manager now.

**Reason for the change**:
Request from LX team, and also to fix the double GA counting.

Closes: https://app.asana.com/0/959084297721550/947351594969261

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -73,12 +73,6 @@ module.exports = {
         ],
       },
     },
-    {
```
Comment (ksigler7) [inline gatsby-config.js]: This looks like we're removing GA from the whole site?

Comment (richbachman) [reply gatsby-config.js]: @ksigler7 We're just removing the GA Gatsby plugin. We're keeping the Google Tag Manager plugin, which loads GA via Google Tag Manager.

Comment (ksigler7) [reply gatsby-config.js]: cool - thanks for the explanation

**Comment by thinkingserious** [comment]
Hello @richbachman,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Comment by ksigler7** [comment]
@richbachman - once we deploy this, are we good here? When we talked about this before, you said there might be something we need to go through and add to the MD?

**Comment by richbachman** [comment]
@ksigler7 yeah we should be good to go.

**Pull request merged**